tests/_runner.lua

```diff
@@ -65,24 +111,17 @@ runner.run_steps = function(steps, options)
         io.flush()  -- for "tail -f".
         step_count = step_count + 1
         local step_as_string = step..'/'..#steps..' (@'..step_count..')'
-        if verbose then
-            io.stderr:write(string.format('Running step %s..\n', step_as_string))
-        end
+        runner.verbose(string.format('Running step %s..\n', step_as_string))
 
         -- Call the current step's function.
         local success, result = xpcall(function()
             return steps[step](step_count)
         end, debug.traceback)
 
         if not success then
-            io.stderr:write('Error: running function for step '
-                            ..step_as_string..': '..tostring(result)..'!\n')
+            runner.done('running function for step '
+                            ..step_as_string..': '..tostring(result)..'!')
```

Comment:
Wrap this into a single line / or align the continuation line?!

Reply:
Both of these do not fit into the textwidth=80, so I just left the indentation the way it was before. Only now do I understand that this was meant to be aligned to the `(` in the line before...

Will handle this later.